presto-main/src/main/java/com/facebook/presto/server/HttpRequestSessionContext.java

```diff
@@ -502,7 +530,7 @@ public Optional<Tracer> getTracer()
      */
     private boolean isTracingEnabled()
     {
-        String clientValue = systemProperties.getOrDefault(DISTRIBUTED_TRACING_MODE, TracingConfig.DistributedTracingMode.NO_TRACE.name());
+        String clientValue = systemProperties.getOrDefault(DISTRIBUTED_TRACING_MODE, "");
```

Comment:
When the property isn't set by the user, it will allow the system default to be used instead of defaulting to no trace. It should be a nice feature to have so applications do not need to change their presto client request to enable tracing, but only need to change their presto application properties.